examples/postgres/main_test.go

```diff
@@ -0,0 +1,1021 @@
+// Copyright 2025 The Go MCP SDK Authors. All rights reserved.
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file.
+
+package main
```

Comment:
Tests that are only about PostgresServer should move to postgres_server_test.go.